truthbeam/config.go

```diff
@@ -19,5 +23,9 @@ func (cfg *Config) Validate() error {
 	if cfg.ClientConfig.Endpoint == "" {
 		return errors.New("endpoint must be specified")
 	}
+	// Normalize cache TTL: 0 means no expiration (same as -1/NoExpiration)
+	if cfg.CacheTTL == 0 {
```

Comment:
Here ([‎truthbeam/internal/metadata/testdata/config.yaml](https://github.com/complytime/complytime-collector-components/pull/45/changes#diff-76035e0368f992ab598134ad1a78fdd961ca904281fe59353bc95822e071faf2) ) multiple formats are expected. It would be good to include unit tests for these inputs.